content/intro-curriculum/data/data_workflow.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Earthquakes and water tables"
+author: "Lindsay R. Carr"
+date: "October 5, 2017"
+output: html_document
+---
+
+The following is an example of data analysis workflow borrowed from the blog "Groundwater levels across US affected by Mexico earthquake" by Laura DeCicco (visit that blog post [here](https://owi.usgs.gov/blog/earthquake/)). Note that your plots might differ slightly from the blog post. The blog was created while the data was still provisional and the data may have been adjusted during review.
+
+## First, get data.
+
+```{r get-data, warning=FALSE, message=FALSE}
+library(dataRetrieval)
+
+sites <- c("402411077374801", "364818094185302", "405215084335400", "370812080261901",
+           "393851077343001", "444302070252401", "324307117063502", "421157075535401",
+           "373904118570701", "343457096404501", "401804074432601", "292618099165901")
+
+gw_data <- readNWISuv(sites, 
+                      parameterCd = "72019", 
+                      startDate = "2017-09-07",
+                      endDate = "2017-09-08")
+
+unique_sites <- attr(gw_data, "siteInfo")
+
+```
+
+## Clean, arrange, and prepare data
+
+```{r clean-data, warning=FALSE, message=FALSE}
+library(dplyr)
+
+# order site info data from Southern-most sites to Northern-most
+south_to_north <- arrange(unique_sites, desc(dec_lat_va))
+
+# subset data so it only includes September 8th, 2017
+startDateTime <- as.POSIXct("2017-09-07 00:00:00", tz="UTC")
+endDateTime <- as.POSIXct("2017-09-09 00:00:00", tz="UTC")
+gw_data_sep8 <- gw_data %>%
+  filter(dateTime > startDateTime,
+         dateTime < endDateTime) 
+
+# use the dataRetrieval fxn to get more user-friendly column names
```

Comment:
could spell out `function` to be super-uber noob-friendly

Reply:
great suggestion! Fixed!